db/migrate/20160909092420_remove_size_from_team.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveSizeFromTeam < ActiveRecord::Migration[5.0]
+  def change
```

Comment:
all this migrations in which you have changed other migrations that you haven't deployed to master is better to change them in the original migration. This is because that attribute hasn't been added to the production db normally
or in the case of the dev team that noone else is working on the same branch